documentation/src/main/asciidoc/topics/code_examples/MaxSizeMemory.java

```diff
@@ -0,0 +1,9 @@
+ConfigurationBuilder cfg = new ConfigurationBuilder();
+
+cfg
+  .encoding()
+    .mediaType("application/x-protostream")
+  .memory()
+    .maxSize("1.5GB")
+    .whenFull("REMOVE")
```

Comment:
`EvictionStrategy.REMOVE`. The method does not accept a string, only the `EvictionStategy` enum as an argument.

Reply:
good catch, thanks. I even have that elsewhere in the docs. I should RTFM myself...

Reply:
This is where compiling code examples would really come in handy.

Reply:
indeed. one day... I've made a couple of attempts with jbang but I think I'm going to need someone with better java skillz. (wink, wink).